FullStoryMiddleware/FullStoryMiddleware.m

```diff
@@ -12,6 +12,23 @@
 #import "FullStoryMiddleware.h"
 #import "FSSuffixedProperties.h"
 
+
+// private SEGScreenPayload to provide the signitures to check for during respondsToSelector:@selector
+// this is due to a signiture change between Analytics 3 and 4.
```

Comment:
I just looked again and realized that we're always requiring a version greater than 4.1.2. So I don't think the interface shenanigans are necessary actually?

Reply:
I thought in the pod spec I defined the dependency which pulls whichever version is currently in the application, I would say the desired behavior is to depend on the consumer of this package to decide which Analytics version they are using. 
```
spec.dependency 'FullStory'
spec.dependency 'Analytics'
```

Reply:
Sorry I was looking at the `Podfile`, not the podspec. In that case, we should handle 3.x and 4.x differences.